### PR TITLE
Add a new README_OSX.md with OS X experimental installation instructions.

### PR DESCRIPTION
**Pull request by grempe**
Show people how to use with OS X for development and testing.

Please review and correct any instructions I am suggesting if there is a better way.

### CODE REVIEW

File: README_OSX.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+
+````
+createuser -s -r postgres
+````
+
+### Install Nodejs using [nvm](https://github.com/creationix/nvm)
+
+See the [nvm](https://github.com/creationix/nvm) website for detailed instructions. Crypton currently strictly expects Nodejs version `0.12.x` to be installed (which is no longer the current stable version).
+
```
Comment (daviddahl) [inline README_OSX.md]: Crypton should work with later node.js, we just need to run through it and all dependency hell to test. I'll add an issue for this.

File: README_OSX.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+````
+bin/cli.js run
+````
+
+Option B) Run in the background
+
+````
+bin/cli.js start
+bin/cli.js status
+````
```
Comment (daviddahl) [inline README_OSX.md]: after `npm link` you should be able to do: `crypton-server run` and `crypton-server tail` etc.

File: README_OSX.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+````
+curl --insecure -i https://localhost:1025
+````
+
+Stop the server
+
+````
+bin/cli.js stop
+````
```
Comment (daviddahl) [inline README_OSX.md]: also: crypton-server stop

**Comment by daviddahl** [comment]
LGTM, but the `crypton-server` command should be available globally after `npm link`

**Comment by grempe** [comment]
Added a new commit to use cryptos-server command in place of bin/cli.js

**Comment by Bennyoak** [comment]
Very nice.

**Comment by daviddahl** [comment]
+1

**Comment by grempe** [comment]
Great.  Thanks for merging.

**Pull request merged**